apps/firebase/compile_rules.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+# Compiles firebase rules from rules.yaml to rules.json using bolt.
```

Comment:
Oops. Correct. yaml was the input format for Blaze, which was Firebase's previous attempt (before Bolt) at a language that compiles down to the security rule language.